### PR TITLE
LPS-109391 Also update the total likes before sending the request to the server

### PR DESCRIPTION
**Pull request by ambrinchaudhary**

### CODE REVIEW

File: modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/RatingsLike.es.js [hunk 1]
```diff
@@ -27,17 +27,20 @@ const RatingsLike = ({
...
+	const [liked, setLiked] = useState(isLiked);
 	const [totalLikes, setTotalLikes] = useState(positiveVotes);
 
 	const toggleLiked = () => {
-		sendVoteRequest(liked ? SCORE_UNLIKE : SCORE_LIKE);
+		const score = liked ? SCORE_UNLIKE : SCORE_LIKE;
 
 		setLiked(!liked);
+		setTotalLikes(totalLikes + score);
+		sendVoteRequest(score);
```
Comment (boton) [inline modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/RatingsLike.es.js]: ❤️

**Comment by 4lejandrito** [comment]
Just started reviewing :)

**Comment by 4lejandrito** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/85756. [See changes here.](https://github.com/4lejandrito/liferay-portal/compare/174437fb3f5c~6...174437fb3f5c)

**Pull request closed without merging**